deploy/helm/kubecf/values.yaml

```diff
@@ -263,6 +263,15 @@ settings:
 
 
 features:
+  # To support multi-clusters, the control_plane is used to deploy cf components except diego-cell. 
+  # While the cluster_worker is to enable diego-cell. When control plane feature is enabled, the cluster-worker cannot be enabled.
+  control_plane:
+    enabled: false
+    workers:
+    - name: minions-1-diego-cell-0
+      address: 172.30.6.59
+  cluster_worker:
```

Comment:
Shouldn't this match this?

https://github.com/cloudfoundry-incubator/kubecf/pull/1107/files#diff-8fc2de7ef5a3d25ddfce0cd023890d0cR251

Isn't the `cluster_worker` the same as a `cell_segment`?

Reply:
@viovanov  I have updated the features to sync with the PR #1107 .Please take a look. Thanks